infobip-typescript-generator-extension-model-showcase/src/main/java/com/infobip/typescript/showcase/custom/complex/ComplexValidationTypeScriptFileGenerator.java

```diff
@@ -0,0 +1,48 @@
+package com.infobip.typescript.showcase.custom.complex;
+
+import com.infobip.typescript.CustomValidationSettings;
+import com.infobip.typescript.TypeScriptFileGenerator;
+import com.infobip.typescript.showcase.custom.complex.validation.Foo;
+import cz.habarta.typescript.generator.Input;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class ComplexValidationTypeScriptFileGenerator extends TypeScriptFileGenerator {
+
+    public ComplexValidationTypeScriptFileGenerator(Path basePath) {
+        super(basePath);
+    }
+
+    @Override
+    public Input getInput() {
+        return Input.from(Foo.class);
+    }
+
+    @Override
+    public Path outputFilePath(Path basePath) {
+        Path lib = basePath.getParent().getParent().resolve("dist");
+
+        try {
+            Files.createDirectories(lib);
```

Comment:
can you put lines 26 and 34 inside try block?

Reply:
done